microcosm_flask/conventions/encoding.py

```diff
@@ -94,11 +97,7 @@ def load_query_string_data(request_schema, query_string_data=None):
     if query_string_data is None:
         query_string_data = request.args
 
-    request_data = request_schema.load(query_string_data)
-    if request_data.errors:
-        # pass the validation errors back in the context
-        raise with_context(UnprocessableEntity("Validation error"), dict(errors=request_data.errors))
-    return request_data.data
+    return request_schema.load(query_string_data)
```

Comment:
May want to wrap this here too; none of the tests complained, but that may be because there aren't any.

Reply:
It seems like you should.